### DISCUSSION TITLE
Italian language

### DISCUSSION THREAD

**Discussion by verbaccia** [OneKeyHQ/app-monorepo#6315] [General]
Hi, has I bought a Onekey Classic 1S and I see that italian language Is not present  I would to know of It It possibile have some texts in english to translate in Italian .
I don't know of Is the right Place to ask . If Is It not i Will appreciate any link to contact for offer my help .
Some years ago with my Linux  group , Majalinux, we translate the Python's library.

**Comment by PatrickChoo** [comment]
We will carefully consider adding support for the Italian language in OneKey Classic 1S. 🫡

**Reply by assetgrowth80-arch** [reply]
Hi, yes, I understand what you mean. You’re referring to the Italian localization of the OneKey Classic 1S device interface, not simply translating the OneKey website or documentation.

Thank you for offering to help with the Italian translation. OneKey already has an open-source development repository, and the project includes internationalization/localization work. 

I think your previous experience with Linux and translating Python libraries could be very useful. The best approach would be to contact the OneKey development team or open a discussion/request in the OneKey GitHub project and ask specifically about contributing Italian translations for the Classic 1S firmware/device UI. The Classic 1S firmware also has internationalization support in its development history. 

If you are willing to contribute, you could mention that you can translate and review the English strings into native Italian, and that you would be happy to follow their translation workflow and submit the work for review.

Thank you for taking the initiative to help improve Italian support for OneKey. 🇮🇹🤝
